job/ipxe.go

```diff
@@ -54,7 +54,7 @@ func (j Job) serveBootScript(w http.ResponseWriter, req *http.Request, name stri
 
 	s := ipxe.NewScript()
 	s.Set("iface", j.InterfaceName(0)).Or("shell")
-	s.Set("tinkerbell", "http://"+env.PublicFQDN)
+	s.Set("boots", "http://"+env.PublicFQDN)
```

Comment:
lets use a different name here, not quite sure what though.

Reply:
What should it be changed to then?